spec/support_spec.rb

```diff
@@ -26,4 +26,10 @@
       expect($LOAD_PATH).to include(dirb)
     end
   end
+
+  describe "#ref" do
+    it 'should return a new RawString with the type/title format' do
+      expect(subject.ref('Package','tomcat').inspect).to eq("Package['tomcat']")
```

Comment:
You're testing too much here.

```
it { expect(subject.ref("A", "b").to be_a RSpec::Puppet::RawString }
```

is enough. RawString's behaviour is tested there.